### PR TITLE
List metric support

### PR DESCRIPTION
**Pull request by bhuvnesh17**
To add list and map metrics support

### CODE REVIEW

**Review by pradeep-nair — CHANGES_REQUESTED** [review]
please check review comments

File: src/main/resources/conf/config.yml [hunk 1]
```diff
@@ -80,5 +80,73 @@ mbeans:
...
+        # List Metrics
+        - name: "listOfString.metric one"
+          alias: "List Of String|metric one"
+        - name: "listOfString.metric two"
+          alias: "List Of String|metric two"
+        - name: "listOfString.metric three"
+          alias: "List Of String|metric three"
+
+
+# Note: If the current "metricCharacterReplacer" is not working on your current machine,
```
Comment (prashmeh) [inline src/main/resources/conf/config.yml]: The `metricCharacterReplacer` in the comment should be changed to `metricPathReplacements`.

Comment (bhuvnesh17) [reply src/main/resources/conf/config.yml]: done!

**Review by pradeep-nair — APPROVED** [review]

File: src/main/resources/conf/config.yml [hunk 2]
```diff
@@ -138,7 +138,7 @@ mbeans:
           alias: "List Of String|metric three"
 
 
-# Note: If the current "metricCharacterReplacer" is not working on your current machine,
+# Note: If the current "metricPathReplacements" is not working on your current machine,
```
Comment (pradeep-nair) [inline src/main/resources/conf/config.yml]: remove this comment, no longer valid

File: pom.xml [hunk 3]
```diff
@@ -14,56 +14,33 @@
...
         <dependency>
             <groupId>com.appdynamics</groupId>
             <artifactId>appd-exts-commons</artifactId>
-            <version>2.0.2</version>
-        </dependency>
-        <dependency>
-            <groupId>org.yaml</groupId>
-            <artifactId>snakeyaml</artifactId>
-            <version>1.13</version>
+            <version>2.2.0-SNAPSHOT</version>
```
Comment (pradeep-nair) [inline pom.xml]: change to 2.2.0

**Pull request merged**